### PR TITLE
29-where-are-we-going-to-store-type-of-workspaces

### PR DESCRIPTION
**Pull request by torniko111**
36-clean-solution-folder-files-from-unnecessary-waste
27-fill-out-readme-based-on-the-requirements
29-where-are-we-going-to-store-type-of-workspaces

### CODE REVIEW

File: DataLayer/Models/Map.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ public class Map : BaseEntity
         public bool HasKitchen { get; set; }
         public bool HasConfRoom { get; set; }
         public int OfficeId { get; set; }
+        //public Office Office { get; set; }
```
Comment (apilatau) [inline DataLayer/Models/Map.cs]: remove the commented code

File: DataLayer/Models/Report.cs [hunk 2]
```diff
@@ -4,10 +4,15 @@ namespace DataLayer.Models
 {
     public class Report : BaseEntity
     {
-        public int EmployeeId { get; set; }
+        public int UserId { get; set; }
+        public User User { get; set; }
         public int CountryId { get; set; }
+        //public Country Country { get; set; }
```
Comment (apilatau) [inline DataLayer/Models/Report.cs]: remove commented code

**Review by apilatau — APPROVED** [review]

**Review by torniko111 — COMMENTED** [review]
remmoved comments,

**Comment by torniko111** [comment]
remmoved commented object, afther added nessesery classes objects will be added.

**Pull request merged**